README.md

```diff
@@ -48,3 +48,7 @@ We welcome contributions to the Atlan Agent Toolkit! Please follow these guideli
 6. **Documentation:**
    - Update documentation to reflect your changes.
    - Add comments to your code where necessary.
+
```

Comment:
@abhinavmathur-atlan 
Let's move this to README.md file under `modelcontextprotocol` folder, which contains specific instructions for installations